snomed/com.b2international.snowowl.snomed.core.rest/src/com/b2international/snowowl/snomed/core/rest/SnomedConceptRestService.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2011-2021 B2i Healthcare Pte Ltd, http://b2i.sg
+ * Copyright 2011-2022 B2i Healthcare Pte Ltd, http://b2i.sg
```

Comment:
You did not introduce changes in this file, yet the copyright date moved forward. 😃

Reply:
Yeah, we will anyway in the upcoming feature release, so let's keep it.